tests/run-make/split-debuginfo/Makefile

```diff
@@ -47,7 +47,7 @@ else # Some non-Windows, non-Darwin platforms are not stable, and some are.
 	# - `.dwo` never created
 	# - `.dwp` never created
 	off:
-		$(RUSTC) foo.rs -g -C $(UNSTABLEOPTS) split-debuginfo=off
+		$(RUSTC) foo.rs -g $(UNSTABLEOPTS) -C split-debuginfo=off
```

Comment:
If this was exercised in non-linux non-darwin non-windows env, this will fail because this expands to

```make
$(RUSTC) foo.rs -g -C -Zunstable-options split-debuginfo=off
```

but it doesn't fail in CI, which means it's not even exercised in non-linux non-darwin non-windows under CI conditions.